InvenTree/common/views.py

```diff
@@ -269,6 +270,15 @@ def get_form_kwargs(self, step=None):
         
         return super().get_form_kwargs()
 
+    def get_form(self, step=None, data=None, files=None):
```

Comment:
Good stuff here!

Reply:
this is what I mean by inherited functions - reduce the changes to a minimum